plugins/woocommerce/src/Admin/Features/OnboardingTasks/TaskLists.php

```diff
@@ -166,6 +166,7 @@ public static function init_default_lists() {
 				),
 				'tasks'   => array(
 					'StoreConnect',
+					'BrowseMarketplace',
```

Comment:
IMO, it should come before `StoreConnect`, not after. I will ask Poli for re-confirmation.

Reply:
Got a confirmation, `BrowseMarketplace` first and then `StoreConnect`